htdocs/admin/ticket.php

```diff
@@ -644,7 +648,7 @@
 print '</tr>';
 
 // Email for notification of TICKET_CREATE
-print '<tr class="oddeven"><td>'.$langs->trans("TicketEmailNotificationTo").' ('.$langs->trans("Creation").')</td>';
```

Comment:
Why removing the 
`' ('.$langs->trans("Creation").')`
It here to be clear that this email is used on ticket creation vs the next point
`.' ('.$langs->trans("Responses").')'`
used on email responses.

Reply:
Just `(creation)` in brackets looks unclear to me.
I'll make another proposal to include "creation" in a complete sentence into `TicketEmailNotificationTo`